docs/markdown/Writing Plugins/rules-api/rules-api-installing-tools.md

```diff
@@ -178,10 +186,43 @@ The resulting `Pex` object has a `digest: Digest` field containing the built `.p
 Instead of the normal `Get(ProcessResult, Process)`, you should use `Get(ProcessResult, PexProcess)`, which will set up the environment properly for your Pex to execute. There is a predefined rule to go from `PexProcess -> Process`, so `Get(ProcessResult, Process)` will cause the engine to run `PexProcess -> Process -> ProcessResult`.
 
 `PexProcess` requires arguments for `pex: Pex`, `argv: Iterable[str]`, and `description: str`. It has several optional parameters that mirror the arguments to `Process`. If you specify `input_digest`, be careful to first use `Get(Digest, MergeDigests)` on the `pex.digest` and any of the other input digests.
-[block:callout]
```

Comment:
No changes here other than fixing a runon sentence in "You must define" paragraph.